vscode/npm-package/eslint.config.mjs

```diff
@@ -0,0 +1,34 @@
+import stylisticTs from '@stylistic/eslint-plugin-ts';
+import typescriptEslint from "@typescript-eslint/eslint-plugin";
```

Comment:
should we also add the microsoft graph eslint configuration? (see kiota-typescript or graph TS)

Reply:
https://github.com/microsoft/kiota/issues/6319